From 2a9b48d20d74f65ac708b3bcee6a7e2a61aadf88 Mon Sep 17 00:00:00 2001
From: Karel Zak <kzak@redhat.com>
Date: Thu, 21 Aug 2025 09:59:10 +0200
Subject: [PATCH] include/mount-api-utils: avoid using sys/mount.h

The file is unnecessary and introduces duplicate #define of MS_* macros.

Fixes: https://github.com/util-linux/util-linux/issues/3674
Signed-off-by: Karel Zak <kzak@redhat.com>
Upstream-Status: Backport [https://github.com/util-linux/util-linux/commit/2a9b48d20d74f65ac708b3bcee6a7e2a61aadf88]
---
 include/mount-api-utils.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/mount-api-utils.h b/include/mount-api-utils.h
index 920d6766a..e35524168 100644
--- a/include/mount-api-utils.h
+++ b/include/mount-api-utils.h
@@ -6,7 +6,6 @@
 #define UTIL_LINUX_MOUNT_API_UTILS
 
 #ifdef HAVE_LINUX_MOUNT_H
-#include <sys/mount.h>
 #include <linux/mount.h>
 #include <linux/unistd.h>
 #include <sys/syscall.h>
-- 
2.34.1

